emperor/qiime_backports/parse.py

```diff
@@ -245,8 +240,8 @@ def parse_coords(lines):
         raise QiimeParseError("The line with the vector number was not found"
             ", this information is required in coordinates files")
 
-    lines = map(strip, lines[1:])   #discard first line, which is a label
-    lines = filter(None, lines) #remove any blank lines
+    lines = list(map(strip, lines[1:]))   #discard first line, which is a label
+    lines = [_f for _f in lines if _f] #remove any blank lines
```

Comment:
Should this line and the next line be combined, or would that reduce readability too much?